CMakeLists.txt

```diff
@@ -82,24 +82,24 @@ if (NOT broker_is_subproject)
       endif ()
   endif()
 
-  if (MSVC)
+endif()
+
+if ( MSVC )
     # Allow more sections in object files, otherwise Broker fails to compile.
     set(EXTRA_FLAGS "${EXTRA_FLAGS} /bigobj")
-  else ()
+else ()
     # Increase warnings.
     set(EXTRA_FLAGS "${EXTRA_FLAGS} -Wall -Wno-unused -pedantic")
     # Increase maximum number of instantiations.
     set(EXTRA_FLAGS "${EXTRA_FLAGS} -ftemplate-depth=512")
     # Make sure to get the full context on template errors.
     set(EXTRA_FLAGS "${EXTRA_FLAGS} -ftemplate-backtrace-limit=0")
-  endif ()
+endif ()
+include(RequireCXX17)
 
-  # Append our extra flags to the existing value of CXXFLAGS.
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${EXTRA_FLAGS}")
+# Append our extra flags to the existing value of CXXFLAGS.
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${EXTRA_FLAGS}")
 
-endif()
-
-include(RequireCXX17)
```

Comment:
What's the reasoning for moving this `include` up before the `set` above it? I understand the change to move settings the flags outside of the `if` block they're currently in. But this doesn't seem necessary. Not a big deal if it is, I'm just trying to reduce the changeset to what's truly needed.